methods_and_recursion/bottles_of_beer.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+
+=begin
+def beer_bottle(number_of_bottles)
+  if number_of_bottles == 0
+    puts 'No more bottles of beer on the wall, no more bottles of beer.
+Go to the store and buy some more, 99 bottles of beer on the wall.'
+  else
+    puts "#{number_of_bottles} bottles of beer on the wall, #{number_of_bottles} bottles of beer.
+Take one down and pass it around, #{number_of_bottles-1} bottles of beer on the wall."
+    beer_bottle(number_of_bottles - 1)
+  end
+end
+
+beer_bottle(99)
+=end
+
+ def beer_bottle_using_loop(input)
+ while input != 1
+   puts " #{input} bottles of beer on the wall, #{input} bottles of beer.
+Take one down and pass it around, #{input-1} bottles of beer on the wall."
+   input = input - 1
+
+ end
+
+ if input == 1
+   puts " #{input} bottles of beer on the wall, #{input} bottles of beer.
```

Comment:
What if you made a pluralize method that took the number of bottles and a string as an argument?

    pluralize(5, 'bottle')   # => 5 bottles
    pluralize(1, 'bottle')   # => 1 bottle
    pluralize(0, 'bottle')   # => No more bottles

I think that'd make your code better.